tango_ros_common/tango_ros_native/package.xml

```diff
@@ -12,6 +12,7 @@
   <build_depend>cmake_modules</build_depend>
   <build_depend>dynamic_reconfigure</build_depend>
   <build_depend>glog_catkin</build_depend>
+  <build_depend>image_geometry</build_depend>
```

Comment:
I think you also need to add this package in the CMakeLists file. Inside `find_package` and `catkin_package`